lambda/opt-out-export/main.go

```diff
@@ -168,6 +169,17 @@ func generateAlignmentFileName(now time.Time) (string) {
 	return fmt.Sprintf(fileFormat, date, time)
 }
 
+func getAssumeRoleSession(session *session.Session) (*session.Session, error) {
+	parameterName := fmt.Sprintf("/opt-out-import/dpc/%s/bfd-bucket-role-arn", os.Getenv("ENV"))
+	assumeRoleArn, err := getSecret(session, parameterName)
+
+	if err != nil {
+		return nil, fmt.Errorf("Failed to retrieve assume role arn: %w", err)
+	}
+
+	return newSession(assumeRoleArn)
```

Comment:
To get everything running locally, I had to switch to this:

```suggestion
	if isTesting {
		return getAwsSession()
	} else {
		return newSession(assumeRoleArn)
	}
```

and then add: 
`aws --endpoint-url=http://localhost:4566 ssm put-parameter \
    --name "/opt-out-import/dpc/local/bfd-bucket-role-arn" \
    --value "assume_role_arn_123456789" \
    --type "String"`
and `aws --endpoint-url=http://localhost:4566 ssm get-parameter --name "/opt-out-import/dpc/local/bfd-bucket-role-arn"`
to populate_aws.sh